src/main/java/org/prebid/server/settings/CachingApplicationSettings.java

```diff
@@ -79,22 +84,27 @@ public CachingApplicationSettings(ApplicationSettings delegate, SettingsCache ca
         return getFromCacheOrDelegate(ampCache, requestIds, impIds, timeout, delegate::getAmpStoredData);
     }
 
-    private static <T> Future<T> getFromCacheOrDelegate(Map<String, T> cache, String key, Timeout timeout,
+    private static <T> Future<T> getFromCacheOrDelegate(Map<String, T> cache,
```

Comment:
Just a thought for @rpanchyk to consider - how about we make this private method non-static? 

Because basically 3 out of 5 of its parameters are class fields which could be accessed directly.